packages/stream_video/lib/src/call/state/call_state_notifier.dart

```diff
@@ -22,13 +24,19 @@ class CallStateNotifier extends StateNotifier<CallState>
   CallStateNotifier(CallState initialState) : super(initialState) {
     callStateStream =
         MutableStateEmitterImpl<CallState>(initialState, sync: true);
+    _durationTimerController = StreamController<int>.broadcast();
+    _ensureStartDurationTimer();
   }
 
   final _logger = taggedLogger(tag: 'CallStateNotifier');
 
   late final MutableStateEmitterImpl<CallState> callStateStream;
   CallState get callState => callStateStream.value;
 
+  Stream<int> get durationStream => _durationTimerController.stream.distinct();
```

Comment:
Even though we update once per second I think I would still do the duration as Duration object.